app/src/main/res/layout/activity_main.xml

```diff
@@ -26,9 +26,31 @@
         android:textColor="@color/White"
         android:textStyle="bold"
         android:layout_gravity="center_horizontal"
-        android:padding="35dp"
+        android:padding="25dp"
         android:textSize="48sp" />
 
+<!--    <EditText-->
+<!--        android:id="@+id/lat_edit_text"-->
+<!--        android:layout_width="match_parent"-->
+<!--        android:layout_height="wrap_content"-->
+<!--        android:autofillHints="false"-->
+<!--        android:hint="Latitude (-90 to 90)"-->
+<!--        android:inputType="numberDecimal|numberSigned" />-->
```

Comment:
why is this block of code commented out? lol please remove if not necessary

Reply:
Will remove

Reply:
Done